### ISSUE TITLE
tensorflow.python.framework.errors_impl.NotFoundError: annotations/label_map.pbtxt ; No such file or directory

### ISSUE THREAD

**Issue by Homayra22** [tensorflow/models#9870]

**Comment by Homayra22** [comment]
I am using tf2 for  object detection api.  When I am trying to train my model on a ubuntu based laptop with GPU support, I am constantly getting this error.

**Comment by saikumarchalla** [comment]
@Homayra22 Based on the error above, it looks like the path to the labels file is incorrect.Could you please check the full path of the label_map.pbtxt file on your system.Thanks!

**Comment by Homayra22** [comment]
@saikumarchalla yes, the label path is correct. I have checked it several times in pipeline configuration file. Even I have tried both with relative path and absolute path. But, nothing works. And the labelling ID is also correct in label_map.pbtxt file.

**Comment by chandyalex** [comment]
@Homayra22  Hi a small tip to try, go to the directory in which label_map.pbtxt exists and type "pwd" it will give you the path and compare it with the path you have given.

**Comment by Homayra22** [comment]
@chandyalex Thanks for the comment. Unfortunately,  I have already tried it. :(
